xup/xup.c

```diff
@@ -1358,7 +1358,7 @@ lib_send_client_info(struct mod *mod)
 
     g_writeln("lib_send_client_info:");
     make_stream(s);
-    init_stream(s, 8192);
+    init_stream(s, (sizeof(mod->client_info))+0xFFF & ~0xFFF);
```

Comment:
the structure became bigger that the magic value of 8192 resulted in xrdp crashes.